practice.js

```diff
@@ -0,0 +1,63 @@
+
+const capitalize = (string) => {
+  return string[0].toUpperCase() + string.slice(1);
+}
+
+const reverseString = (string) => {
+  return string.split('').reverse().join('');
+}
+
+const calculator = {
+  add: (a, b) => {
```

Comment:
[OPTIONAL]
We can write these simply like `add: (a, b) => a + b` as we only have a single return statement in the arrow function.